app/controllers/concerns/wallet_func.rb

```diff
@@ -0,0 +1,18 @@
+module WalletFunc
+  def create_random_wallet_number
+    rand(1_111_111_111_111_111..9_999_999_999_999_999)
+  end
+
+  def currencies
+    Currency.all
+  end
+
+  def unlocked_user_wallets(user)
```

Comment:
add specs for this please

Reply:
In which file I need to add test for this method?